.github/label-conflict.yml

```diff
@@ -0,0 +1,40 @@
+name: Auto Label Conflicts
+on:
+  push:
+    branches:
+      - 'develop'
+      - '4.*'
+  pull_request:
+    branches:
+      - 'develop'
+      - '4.*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  auto-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: prince-chrismc/label-merge-conflicts-action@v3
+        with:
+          conflict_label_name: "stale"
+          github_token: ${{ github.token }}
+
+          # --- Optional Inputs ---
+          # To make sure the merge commit exactly matches the branch
+          detect_merge_changes: true # or true to handle as conflicts
+          # By default a comment will be left, adding `conflict_comment: ''` will disable comments
+          # The optional `${author}` will be replaced with the username of the pull request
+          conflict_comment: |
+            :wave: Hi, @${author},
+            We detected conflicts against the base branch :speak_no_evil:
+            You'll want to sync :arrows_counterclockwise: your branch with upstream!
+
+            Ref: https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#pushing-your-branch
```

Comment:
You mean https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch ?

Reply:
Ya, almost similiar. When i forget, i usually read on https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#pushing-your-branch